review-notifications/src/background.js

```diff
@@ -139,7 +144,14 @@ async function findMentioned(commentsUrl) {
 async function extractDataFromPR(prObject) {
   try {
     const mentionedUsers = await findMentioned(prObject.comments_url);
-    const comments = await extractComments(prObject);
+    const reviewComments = await extractComments(prObject.review_comments_url);
+    const prComments = await extractComments(prObject.comments_url);
+    const comments1 = [].concat.apply([], reviewComments.data);
+    const comments2 = [].concat.apply([], prComments.data);
+    const comments = comments1.concat(comments2);
```

Comment:
It's alright, can also be written using spread operators `[...comments1, ...comments2]`